help/en/html/tools/logixng/reference/chapter9.shtml

```diff
@@ -303,6 +303,40 @@
 
       <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
 
+      <h3>Access to Java Methods</h3>
+
+      <p>Sometimes it is possible to access Java methods.  For example, for most of the JMRI beans
+      it is possible to access the public methods in the bean's Interface.  This can also apply to
+      some of the main Java classes, such as String.</p>
+
+      <h5>Java Arrays<span class="since">since 5.7.5</span></h5>
+
+      <p>Some Java methods return a Java Array instead of a List.  The LogixNG Formula has been
+      enhanced to translate a Java Array result into a LogixNG Array, which is actually a List.</p>
```

Comment:
The LogixNG formula can now handle a Java array the same way as it handles a list. But it doesn't translate the Java Array to a list.

Reply:
Done.

However, is it possible to get the number of rows in the array?  "For each" does not work.  And the "for" loop needs the row count.

Reply:
> However, is it possible to get the number of rows in the array? "For each" does not work. And the "for" loop needs the row count.

It's fixed with PR #12942.